src/Tools/Source/RunTests/TestRunner.cs

```diff
@@ -156,29 +157,47 @@ string makeHelixWorkItemProject(AssemblyInfo assemblyInfo)
 
                 var commandLineArguments = _testExecutor.GetCommandLineArguments(assemblyInfo, useSingleQuotes: isUnix);
                 commandLineArguments = SecurityElement.Escape(commandLineArguments);
+                var setEnvironmentVariable = isUnix ? "export" : "set";
+
+                var command = new StringBuilder();
+                command.AppendLine(isUnix ? "ls -l" : "dir");
```

Comment:
I have considered eliminating these commands or just compressing them down into "we had x files before rehydration and y files afterward". They're good for debugging but when people just want to see output from the test runner it kinda gets in the way.

Reply:
I guess in my case I wasn't sure if the script was going to have the execute bit set, this was good to also just confirm this.

My motto is if we're regularly having to look at the full log, that means we need better tooling somewhere else.

Reply:
fair enough!